bindings/azure/blobstorage/blobstorage.go

```diff
@@ -156,7 +160,17 @@ func (a *AzureBlobStorage) create(blobURL azblob.BlockBlobURL, req *bindings.Inv
 		Metadata:        req.Metadata,
 		BlobHTTPHeaders: blobHTTPHeaders,
 	})
-	return nil, err
+	resp := createResponse{
+		BlobURL: blobURL.String(),
+	}
+	b, err := json.Marshal(resp)
+	if err != nil {
+		return nil, fmt.Errorf("error marshalling create response for azure blob: %s", err)
```

Comment:
I am not sure we start using pkg/errors pkg here. do we ? then 
I would use `errors.Errorf` instead of fmr.Errorf

Reply:
We already use it in other places, but not globally. Introducing a change here will need a change in the entire file. What do you think?

Reply:
I am ok with either way. we can do everything later.